### PR TITLE
[Reply, Econews] Incorrect alignment of 'Редактировать' label on the button #1629

### PR DESCRIPTION
**Pull request by romanovskyroma**
corrected @media, removed the width from the buttons.
[1629](https://github.com/ita-social-projects/GreenCity/issues/1629)

### CODE REVIEW

**Review by luchward — COMMENTED** [review]
This is looking as described bug was eliminated. However I notised the same bug in Ukrainian localisation. Would you think about its elimination?
Opera Знімок_2020-10-18_223809_localhost

**Comment by romanovskyroma** [comment]
> This is looking as described bug was eliminated. However I notised the same bug in Ukrainian localisation. Would you think about its elimination?
> Opera Знімок_2020-10-18_223809_localhost

the bug in the screenshot is a bug under a different number

File: src/app/component/comments/components/delete-comment/delete-comment.component.scss [hunk 1]
```diff
@@ -1,14 +1,13 @@
 @import url('https://fonts.googleapis.com/css?family=Open+Sans&display=swap');
 
-$main-font: 'Open Sans', sans-serif;
+$main-font: 'Open Sans',
+  sans-serif;
```
Comment (yulkats) [inline src/app/component/comments/components/delete-comment/delete-comment.component.scss:4]: Why do you move to new line?

**Review by yulkats — APPROVED** [review]

**Review by skrebetsnataliia — APPROVED** [review]

**Review by HryshkoAnton — APPROVED** [review]

**Comment by romanovskyroma** [comment]
bug already fixed

**Review by mr-pavlenko — APPROVED** [review]

**Pull request merged**